bot/client.js

```diff
@@ -15,6 +15,14 @@ client.on('ready', () => {
   util.log('Bot Online and Ready', 0);
 });
 
+// Message for new users
+client.on('guildMemberAdd', (member) => {
+  // Send Direct Message with Terms of Service
+  member.send(`Welcome ${member} to the Full Sail Armada! Read our Terms of Service`);
```

Comment:
Duplicate code logic in your controller.

Reply:
If I understand correctly you are talking about the direct message here being the same as the direct message in my tos.js tosAction. I see that I could fire off the tosAction inside here but that would require me to import my specific controller, TosController. Is this your preferred way to solve the issue?